core/src/replay_stage.rs

```diff
@@ -3440,8 +3440,16 @@ impl ReplayStage {
         // 3) The best "selected" bank is on a different fork,
         //    switch_threshold succeeds
         let mut failure_reasons = vec![];
-        let selected_fork = {
-            let switch_fork_decision = tower.check_switch_threshold(
+        struct CandidateVoteAndResetBanks<'a> {
+            // A bank that the validator will consider voting on given it passes all
+            // remaining vote checks
```

Comment:
nit: would it better to be: A bank that validator will vote on if it passes all remaining vote checks ?

If it passes all checks below it will be the voting bank, there is nothing else to consider, right?

Reply:
true, changed